modules/@ergonode/core/src/components/Buttons/ExpandNumericButton.vue

```diff
@@ -42,11 +43,16 @@ export default {
     },
     props: {
         /**
-         * Determines state of expanded component
+         * The size of the component
          */
-        isExpanded: {
-            type: Boolean,
-            default: false,
+        size: {
+            type: String,
+            default: SIZE.SMALL,
+            validator: value => [
+                SIZE.TINY,
+                SIZE.SMALL,
+                SIZE.REGULAR,
+            ].indexOf(value) !== -1,
```

Comment:
that's shorter : `validator: value => Object.values(SIZE).indexOf(value) !== -1,` and if we change the defaults size, we don't have to change it everywhere.

Reply:
We do not want to allow for each SIZE to be passed